developer_tools/stratis_migrate_symlinks.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Migrate the symlinks of Stratis filesystems from the old
+# /stratis path to /dev/stratis, via synthetically generating
+# udev change events.
+
+if [ ! -d /stratis ]
+then
+	echo "'/stratis' directory does not exist; nothing to do."
+	exit 0
+fi
+
+if [ ! -x /usr/lib/udev/stratis_uuids_to_names ]
+then
+	echo "stratis_uuids_to_names' udev program does not exist."
+	exit 2
+fi
+
+if [ ! -x /usr/bin/stratis_dbusquery_version ]
+then
+	echo "'stratis_dbusquery_version' program does not exist."
+	exit 2
+fi
+
+if [ ! -f /usr/lib/udev/rules.d/60-stratisd.rules ]
```

Comment:
This will need to be updated to the prefix 11.

Reply:
Resolved.